packages/decoder/lib/decoders.ts

```diff
@@ -45,7 +46,7 @@ const SourceMapUtils = require("@truffle/source-map-utils");
  * @category Decoder
  */
 export class ProjectDecoder {
-  private web3: Web3;
+  private web3Adapter: Web3Adapter;
```

Comment:
Does this need to be public?

Reply:
Doesn't seem like it - why might it need to be public?

Reply:
Doesn't seem like it - why might it need to be public?

Reply:
Yeah there's already a method to get it...